corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html

```diff
@@ -79,6 +79,7 @@
   {% initial_page_data 'single_app_mode' single_app_mode %}
   {% initial_page_data 'username' username %}
   {% initial_page_data 'use_live_query' use_live_query %}
+  {% initial_page_data 'has_geocoder_privs' has_geocoder_privs %}
```

Comment:
Have you tested in both web apps and app preview? I think app preview uses a different base template.